frontend/amundsen_application/static/js/components/Alert/AlertList.tsx

```diff
@@ -10,19 +10,46 @@ export interface AlertListProps {
   notices: NoticeType[];
 }
 
+export interface AggregatedAlertListProps {
+  notices: {
+    [key: string]: NoticeType;
+  };
+}
+
+const aggregateNotices = (notices) =>
+  notices.reduce((accum, notice: any) => {
+    if (notice) {
+      const { messageHtml, severity, payload } = notice;
+
+      if (payload) {
+        accum[messageHtml] ??= {
```

Comment:
I just noticed this in your screenshot in the PR description where it groups the notices with the same message but different severities into 1 - I'm thinking if the severity is different we should group those separately if possible, so by unique message and severity? I don't think that would happen for us in our cases, but not sure if it might for others who use this feature

Reply:
Ah this is a great point. Yeah I can group them by message + severity, let me get a fix up. Thanks for the find!

Reply:
Alright, took a few days to get back to this but the changes are now up and screenshots are updated. So it will now group by message and then sub-group by severity, which is then sorted to show up